cli/command/load_balancer.go

```diff
@@ -217,13 +218,14 @@ func (l *LoadBalancerCommand) create(c *cli.Context) error {
 		ports = append(ports, port)
 	}
 
-	target, err := parseTarget(c.String("healthcheck-target"))
-	if err != nil {
-		return err
+	if c.String("healthcheck-target") != "" {
```

Comment:
what is `parseTarget` being used for? If it's just validation, we don't need to do that here since aws takes care of that for us. If we decide to keep it, `validateTarget` is probably a more appropriate name (and it shouldn't return the target since it doesn't manipulate it in any way)

Reply:
It is just for validation in this case.